api/managementpb/dbaas/psmdb_cluster.proto

```diff
@@ -70,12 +72,27 @@ message ListPSMDBClustersResponse {
     // Currently running operation, if any.
     RunningOperation operation = 3;
     // Cluster parameters.
-    PSMDBClusterParams params = 4;
+    PSMDBClusterParamsList params = 4;
   }
   // PSMDB clusters information.
   repeated Cluster clusters = 1;
 }
 
+// PSMDBClusterParamsList represents PSMDB cluster parameters that can be listed.
```

Comment:
As far as I see this message doesn't have difference from `PSMDBClusterParams`

Reply:
So maybe we can revert this changes.

Reply:
Yes, right - wil do.

Reply:
Reverted.